src/test/ui/rfc-2008-non-exhaustive/struct.stderr

```diff
@@ -62,18 +62,33 @@ error[E0638]: `..` required with struct marked as non-exhaustive
    |
 LL |     let NormalStruct { first_field, second_field } = ns;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+help: add `..` at the end of the field list
+   |
+LL |     let NormalStruct { first_field, second_field , .. } = ns;
+   |                                                  ^^^^^^
```

Comment:
I'd like it if we could get around pointing at the closing brace and the preceding space, but that would complicate the span gather code quite dramatically...

Reply:
Trust me, I tried several clever iterations to fix that but I got nowhere. :)